### PR TITLE
Register asteral.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Register asteral.is-a.dev with CNAME record pointing to asteral1.github.io.

### CODE REVIEW

File: domains/asteral.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Asteral1",
+           "email": "",
+           "discord": "870366927653056582",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.F2v6VwlZy8PN3eAm0sdM9Kqvh7Cah2AqvxrUMBqNmUI0p-bZbD4Ovfjm2Ti_5SOJB2CF_Ff-2Fb1oaJfef7OeUL9YIdu2O2jFqUR81znaNuPoT9VpXEIAeIQwBvQylSRqnhZK92QALGNUOIo8_9scBEZ-4K1Sfgn6ypISO-O5en13fcYW5gjX_CdAy1CjzHU7PZ0EuF-Ym0G5FXj6gGY3AbysoSDAef38r7uzFbUKEbcRd6QUjMoQkGp9ST9NqP_-Mg0XFEPC9Bbyc6lvGwe2DbIoNzUqlo3b6nViKSiS0UIXe_cT0aImPRqvR4N5wSvwR3XK1R2mY0EewgsTjq7vg.7AhkpT73ke6dCcN9XNwSMw.E_L4WPiryeVCW_0jWiWMA6QjWu5gIIA4Ipysar9kOANIogUjr8Gwfxrj-0rW9h7gfeSbFaeQdNWKPAwlnkYIzB3Na6nUALnyFY-ki1euFGU.F3Zs36mJlugS4fB-6CtqbQ"
+        },
+    
+        "record": {
+            "CNAME": "asteral1.github.io"
```
Comment (DEV-DIBSTER) [inline domains/asteral.json:10]: 404 GitHub page. Please have content or send a preview.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/asteral.json (hunk 1 above)
Comment (Asteral1) [reply domains/asteral.json:10]: Screenshot_2024-10-06-01-34-14-365_com android chrome

Comment (Asteral1) [reply domains/asteral.json]: Done!

Comment (orxngc) [reply domains/asteral.json]: no, not done, it's still a 404

Comment (Asteral1) [reply domains/asteral.json]: I updated the asteral.json

Comment (Asteral1) [reply domains/asteral.json]: The site can be accessed at https://asteral.github.io/asteral.github.io

**Review by ghost — COMMENTED** [review]
Updated

File: domains/asteral.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Asteral1",
+           "email": "",
+           "discord": "870366927653056582",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.F2v6VwlZy8PN3eAm0sdM9Kqvh7Cah2AqvxrUMBqNmUI0p-bZbD4Ovfjm2Ti_5SOJB2CF_Ff-2Fb1oaJfef7OeUL9YIdu2O2jFqUR81znaNuPoT9VpXEIAeIQwBvQylSRqnhZK92QALGNUOIo8_9scBEZ-4K1Sfgn6ypISO-O5en13fcYW5gjX_CdAy1CjzHU7PZ0EuF-Ym0G5FXj6gGY3AbysoSDAef38r7uzFbUKEbcRd6QUjMoQkGp9ST9NqP_-Mg0XFEPC9Bbyc6lvGwe2DbIoNzUqlo3b6nViKSiS0UIXe_cT0aImPRqvR4N5wSvwR3XK1R2mY0EewgsTjq7vg.7AhkpT73ke6dCcN9XNwSMw.E_L4WPiryeVCW_0jWiWMA6QjWu5gIIA4Ipysar9kOANIogUjr8Gwfxrj-0rW9h7gfeSbFaeQdNWKPAwlnkYIzB3Na6nUALnyFY-ki1euFGU.F3Zs36mJlugS4fB-6CtqbQ"
+        },
+    
+        "record": {
+            "CNAME": "asteral1.github.io/asteral"
```
Comment (orxngc) [inline domains/asteral.json:10]:
```suggestion
            "CNAME": "asteral1.github.io"
```

CNAMEs cannot contain slashes (`/`).

File: domains/asteral.json
Comment (orxngc) [inline domains/asteral.json:1]: This site is incomplete, please add some content. E.g. a "Hi, I'm orangc" will suffice as complete.

**Pull request closed without merging**